README.md

```diff
@@ -14,9 +14,9 @@ To find out if your company's tool is feasible for this approach, check out our
 We provide reference implementations that provide fully functional integrations with popular service desks. 
 These implementations, while functional, are examples only, and have not been vetted for production use.
 
-- [Genesys Cloud](./src/middleware/genesys)  
```

Comment:
Lets add the generic example here, too. And lets combine the example and template folders into just example. All we have to do is pull over `serviceDeskTemplate.ts` into the example folder and add a note in the README.md of that folder (and in the comments of buildEntry) that you can simply just point the buildEntry to the template instead of the example to build your own. See notes below on what else should be in this README.md.